src/EventSubscriber/StrawberryfieldEventInsertSubscriberDepositDO.php

```diff
@@ -20,6 +20,11 @@ class StrawberryfieldEventInsertSubscriberDepositDO extends StrawberryfieldEvent
 
   use StringTranslationTrait;
 
+  /**
+   * Default path for digital object file storage.
+   */
+  public const DEFAULTOBJECTSTORAGEFILEPATH = 'dostorage';
```

Comment:
really like this. Could you add a few "_" to make it more legible? Sorry. It really helps

Reply:
Better now, I hope!